### PR TITLE
GDAL 2.1.2 transition + fixes

### PR DESCRIPTION
**Pull request by kalxas**

### CODE REVIEW

File: bin/install_grass.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ TMP_DIR=/tmp/build_grass
 mkdir "$TMP_DIR"
 
 apt-get --quiet update
-apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal20-2.1.0-grass
+apt-get --yes install grass-core grass-gui grass-doc grass-dev libgdal20-2.1.2-grass
```
Comment (sebastic) [inline bin/install_grass.sh]: You should change `libgdal20-2.1.2-grass` to `libgdal-grass`, see: https://trac.osgeo.org/osgeo/ticket/1776

Comment (kalxas) [inline bin/install_grass.sh]: Thanks @sebastic

Comment (kalxas) [inline bin/install_grass.sh]: https://github.com/OSGeo/OSGeoLive/pull/146/commits/ecb28f7bc900662df9901de09fcaeba7c775da08

Comment (kalxas) [inline bin/install_grass.sh]: Included in the next PR :)

**Pull request merged**